.github/workflows/plugin_update.yml

```diff
@@ -0,0 +1,54 @@
+name: Plugins Update file test
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # run daily at midnight
+
+jobs:
+  check-plugins:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      
+      - name: Configure git & gh
+        run: |
+          
+          echo ${{ secrets.GH_TOKEN }} | gh auth login --with-token
+          gh auth status
+          git config --global user.email "ash.191245141@gmail.com"
+          git config --global user.name "ash-sxn"
+
+      - name: Check plugins
+        run: |
+          # Set the name of the branch to create the pull request from
+          BRANCH_NAME="update-plugins"
```

Comment:
Do you mean you have a special branch that hosts a dedicated `docker-compose.yml` file?
Why do you need this branch? 🤔 
Thanks.